modules/user/user.controller.js

```diff
@@ -11,6 +11,31 @@ const { authenticateJWT } = require(".//auth.jwt.middleware");
 router.use(express.json());
 router.use(express.urlencoded({ extended: true }));
 
+async function extractQueryParamsFromRequest(request) {
```

Comment:
Seems like this function couldn't be used anywhere except _get all users endpoint_, so there is no reason to put it in extra function. It will be better to write this logic directly in controller function.